### ISSUE TITLE
Thrustamaster T300RS Support?

### ISSUE THREAD

**Issue by Dokman** [renzuzu/renzu_steeringwheel#1]
May can i help you to add the support

**Comment by renzuzu** [comment]
yes, do you have that? i only have g29 thats why thats the only thing i added.

**Comment by Dokman** [comment]
I have the T300RS if you want I can help you

**Comment by MCS-1999** [comment]
Hello we have tried to setup other wheel supports and we kinda Got it working our biggest issue is the wheel never stays center [Demo](https://cdn.discordapp.com/attachments/1071253544964980868/1087618047797362788/FiveM_by_Cfx.re_-_No_Hesi_RP___Development_Server_2023-03-21_01-04-59.mp4) please hmu on my discord or here would be great! MCS#5567 the wheel I question was a Moza R5 Belt Driven

**Comment by renzuzu** [comment]
try playing with the client.

since the axis logic is for g29 logitect.

**Comment by MCS-1999** [comment]
So we have configured the axis based off the api you're using and using a game pad emulator to detect the Axis to be able to determine what axis is what while Debugging the js we see the axis when centered fluxes left and right like described in the video above

**Comment by MCS-1999** [comment]
Could it be the wheel itself the wheel just never stays at 0.0 we have even implemented a deadzone and smoothing system which helped but didn't exactly fix it

**Comment by renzuzu** [comment]
my guess is the deadzone of wheel does not match in the current logic from client

![image](https://user-images.githubusercontent.com/82306584/226610893-ad6667be-d157-4454-9243-3ffb38e6a886.png)

this is the hardcoded for g29 probably all logitech the same.

**Comment by renzuzu** [comment]
and this

![image](https://user-images.githubusercontent.com/82306584/226611248-90b38a24-225a-45e2-8719-528e1cb364f4.png) from JS.

because g29 api sends a inverted value for SteerBias Native

**Comment by MrColdStiX** [comment]
Interesting, his wheel did the same as well, left is 1.0 and right is -1.0

**Comment by Gameadictive** [comment]
Why is not working?¿

**Comment by renzuzu** [comment]
only works with g29, make sure the logitech ghub is installed

and when you restart the resource, it should says its detected
